v2/cmd/functional-test/testcases.txt

```diff
@@ -0,0 +1,24 @@
+127.0.0.1 {{binary}} -silent -p 8000
+127.0.0.1 {{binary}} -silent -il
+127.0.0.1 {{binary}} -silent -l test-data/request.txt -p 8000
+127.0.0.1 {{binary}} -silent -p 8000,443
+127.0.0.1 {{binary}} -silent -tp 100 -p 8000
+127.0.0.1 {{binary}} -silent -ep 200  -p 8000
+127.0.0.1 {{binary}} -silent -ec 100,400  -p 8000
```

Comment:
Please remove the double space between CLI args

Reply:
Done

Reply:
Done